lib/cli/commands/addcurrency.ts

```diff
@@ -0,0 +1,38 @@
+import { Arguments } from 'yargs';
+import { callback, loadXudClient } from '../command';
+import { AddCurrencyRequest } from '../../proto/xudrpc_pb';
+import { SwapClients } from '../../types/enums';
+
+export const command = 'addcurrency <currency> <swap_client> [decimal_places] [token_address]';
+
+export const describe = 'add a supported currency';
```

Comment:
`add a supported currency` sounds like you add an already supported currency. What about `add support for a currency` or simply `add a currency`?

Reply:
I'll also change the comments in the proto definition.